content/en/news/releases/1.13.x/announcing-1.13/upgrade-notes/index.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Upgrade Notes
+description: Important changes to consider when upgrading to Istio 1.13.0.
+weight: 20
+---
+
+{{< warning >}}
+This is an automatically generated rough draft of the release notes and has not yet been reviewed.
+{{< /warning >}}
+
+When you upgrade from Istio 1.12.x to Istio 1.13.0, you need to consider the changes on this page.
+These notes detail the changes which purposefully break backwards compatibility with Istio 1.13.0.
+The notes also mention changes which preserve backwards compatibility while introducing new behavior.
+Changes are only included if the new behavior would be unexpected to a user of Istio 1.12.x.
+
+## Health Probes will no longer re-use connections
+
+Health probes using the istio-agent [health probe rewrite](https://istio.io/latest/docs/ops/configuration/mesh/app-health-check/) will
+now no longer re-use connections for the probe. This behavior was changed to match probing behavior of Kubernetes',
+and may also improve probe reliability for applications using short idle timeouts.
+
+As a result, your application may see more connections (but the same number of HTTP requests) from probes.
+For most applications, this will not be noticeably different.
+
+If you need to revert to the old behavior, the `ENABLE_PROBE_KEEPALIVE_CONNECTION=true` environment variable in the proxy may be set.
+
+## Multicluster Secret Authentication Changes
+
+When kubeconfig files are created to [enable endpoint discovery](https://istio.io/latest/docs/setup/install/multicluster/multi-primary/#enable-endpoint-discovery)
+in multicluster installations, the authentication methods allowed in the configuration is now limited.
+
+The two authentication methods output but `istioctl create-remote-secret` (`oidc` and `token`), are not impacted.
+As a result, only users that are creating custom kubeconfig files will be impacted.
```

Comment:
could be useful to explain this change, is it more secure?

Reply:
Yes, this is a security fix.

Reply:
Revised a sentence to explain the change is to improve security: "the authentication methods allowed in the configuration is now limited to improve the security."